### DISCUSSION TITLE
LED round small input meter not looking good

### DISCUSSION THREAD

**Discussion by 38github** [jamulussoftware/jamulus#2491] [UI Related]
I like the look of the LED round small meters but I find the input meter which is large to be robbing the GUI of a nice look. I personally would like it to be exactly like the user meters.

**Comment by pljones** [comment]
I think we should revert to the idea of having two separate settings - choose input meter style and choose mixer style.

(Oh, and just to be clear on your discussion topic, you mean the large round LEDs are not looking good, right - i.e. they'd be better small?)

**Reply by 38github** [reply]
Yes, I think they would best looking and unison if it is the same size as the mixer LEDs.
